docs/architecture/adr-028-public-key-addresses.md

```diff
@@ -208,20 +208,29 @@ If we want to create an address for a module account depending on more than one
 btcAtomAMM := address.Module("amm", btc.Addrress() + atom.Address()})
 ```
 
-We can continue the derivation process and can create an address for a submodule account.
+#### Derived Addresses
 
-```
-func Submodule(address []byte, derivationKey []byte) {
-    return Hash("module", address + derivationKey)
+Module address is a special case of more general _derived_ address defined by the `Derive` function:
```

Comment:
Not sure to get what you mean here, could you rephrase it?

Reply:
In `Derive`, the first argument is `address`. In Module we don't have that argument, but in the implementation we require to use `"module"` for that.